index.bs

```diff
@@ -671,9 +670,9 @@ url: https://w3c.github.io/permissions/#permissions-task-source; type: dfn;
 
 				1. Let |isCustom| be |false|.
 				
-				1. If |key| [=string/starts with=] "web " ("web" followed by U+0020 SPACE) prefix, then
+				1. If |key| [=string/starts with=] `"web "` prefix, then
 				
-					1. Remove "web " prefix and set the remaining string to |key|.
+					1. Remove `"web "` prefix and set the remaining string to |key|.
```

Comment:
nit: `assign the remaining string to |key|` or `set |key| to the remaining string`

Reply:
Done.